docs/pages/api-reference/liveblocks-react-tiptap.mdx

```diff
@@ -80,6 +80,737 @@ Learn more in our [get started guides](/docs/get-started/text-editor/tiptap).
 
 ## Default components
 
+### Toolbar
+
+Displays a toolbar, allowing you to change the styles of selected text. You can
+add JSX before or after, or the toolbar’s options can be
+[customized](#creating-a-custom-toolbar).
+
+[TODO img]
+
+Pass your Tiptap `editor` to use the component. By default, the toolbar has a
+button for creating comment threads, add [`FloatingComposer`][] to enable this.
+Display threads with [`AnchoredThreads`][] or [`FloatingThreads`][].
```

Comment:
I hear this as "By default, this toolbar has a **single** button for creating comment threads", maybe "If Liveblocks Comments are enabled in the editor, the toolbar will also contain a button for creating comment threads..."
We could also tell more about `useLiveblocksExtension({ comments: false })` etc